src/DIRAC/WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -96,10 +75,15 @@ def setPilotCredentials(ce, pilotDict):
     :return: S_OK/S_ERROR
     """
     if "Token" in ce.ceParameters.get("Tag", []):
-        result = getPilotToken(pilotDict)
+        result = gTokenManager.getToken(
+            userGroup=pilotDict["OwnerGroup"],
+            scope=["compute.cancel", "compute.create", "compute.modify", "compute.read"],
```

Comment:
Only hardcoded?

Reply:
Do you have any suggestion?

Reply:
What about this? https://github.com/DIRACGrid/DIRAC/pull/7062/commits/6c2b4f8f8423c02723f32ad9e53e0e293d5504fb